src/renderer/components/Message.js

```diff
@@ -79,8 +118,9 @@ class RenderMessage extends React.Component {
     }
 
     if (msg.attachment && !msg.isSetupmessage) props.attachment = msg.attachment
+    if (message.isInfo) return <InfoMessage>{msg.text}</InfoMessage>
```

Comment:
I would like to have the distinction between the different message rendering types at one place. Currently we're doing it in render() and RenderMessage.render() method. If it's not too much trouble i would prefer to have it at the same place. Otherwise looks good to me :)

Reply:
I think this would need some refactoring and/or rethinking UI, because the whole setup message now is expected to be clickable (outside of the message), otherwise they have to specifically click a box within the text or the text itself to open the setup message.

Reply:
Okay then let's get this merged